playbooks/vendor-evaluations.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Vendor Evaluations
+description: How to evaluate 3rd party vendors
+---
+
+Evaluting vendors can be daunting and overwhelming. Here are list of considerations when evaluation a vendor,
+grouped by larger themes:
+
+**Cost**
+- <details><summary>ROI</summary>
+      <ul>
+          <li>What the value would we get out of integrating with this vendor? Can we put a number on it in terms of dollars/engineering hours? </li>
+      </ul>
+      </details>
+**Ease of integration**
+- <details><summary>Time to delivery</summary>
+    <ul>
+        <li>How long would it take to integrate with this vendor?</li>
+        <li>What is the opportunity cost?</li>
+    </ul>
+    </details>
+- <details><summary>Migration complexity</summary>
+      <ul>
+          <li>What is the path to integration? Is it complex or straightforward?</li>
+          <li>How fleshed out are the docs?</li>
+          <li>Is there a community of devs using this vendor?</li>
+          <li>Does the vendor use well-accepted protocols or their own proprietary implementations?</li>
+          <li>Do we have to replace an in-house solution? How does that affect the integration plan?</li>
+      </ul>
+      </details>
+**Feature set**
+- <details><summary>Does it solve our problem?</summary>
+    <ul>
+        <li>If it is not perfect for our use case, is it good enough?</li>
+        <li>Does the solution scale well? Do you see us continuing using it a year from now? 5 years from now?</li>
+        <li>Does it offer UI/brand customization? Do we need UI/brand customization now? Will we in the future?</li>
+    </ul>
+    </details>
+- <details><summary>Vendor qualities</summary>
+      <ul>
+          <li>Are they the experts in this problem space?</li>
+          <li>What stage of the company is the vendor in? Could that affect integration or support?</li>
+          <li>How fast is the technology or problem space changing? Is the vendor responsive to these changes?</li>
+      </ul>
+      </details>
+**Vendor lockin**
+- <details><summary>Migrating off</summary>
+    <ul>
+        <li>Would to be hard to migrate off this vendor if we change our minds?</li>
+        <li>Is the integration very custom to our stack?</li>
+        <li>Is there a data export feature?</li>
```

Comment:
I would add protocol / data exchange format compatibility here - i.e. does the vendor support established industry standards?